.github/workflows/deploy-to-github-pages.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    name: Deploy to GitHub Pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.15.0
+          cache: npm
+
+      - name: Install dependencies
+        run: npm ci
+      - name: Build website
+        working-directory: ./website
+        run: |
+          npm i
+          npm run build
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

Comment:
I’m surprised that you have to use a 3rd party command when docusaurus provides their own command. Did you try this?

> We provide a docusaurus deploy command that helps you deploy your site from the source branch to the deployment branch in one command: clone, build, and commit.

Reply:
This was part of Docusaru's documentation.   https://docusaurus.io/docs/deployment#triggering-deployment-with-github-actions.

Reply:
This is what was said in the documentation. The original deploy workflow will always be listed as skipped in the PR check suite status, which is not communicative of the actual status and provides no value to the review process. We, therefore, propose to manage them as separate workflows instead.

We will use a popular third-party deployment action: [peaceiris/actions-gh-pages](https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-docusaurus).

Reply:
Works for me